front/app/containers/Admin/projects/project/inputImporter/ImportModal/messages.ts

```diff
@@ -1,15 +1,28 @@
 import { defineMessages } from 'react-intl';
 
 export default defineMessages({
-  inputImporter: {
-    id: 'app.containers.Admin.projects.project.offlineInputs.ImportModal.inputImporter',
-    defaultMessage: 'Input importer',
+  importExcelFileTitle: {
+    id: 'app.containers.Admin.projects.project.offlineInputs.ImportModal.importExcelFileTitle',
+    defaultMessage: 'Import Excel file',
   },
-  uploadAPdfExcelFile: {
-    id: 'app.containers.Admin.projects.project.offlineInputs.ImportModal.uploadAPdfExcelFile',
+  importPDFFileTitle: {
+    id: 'app.containers.Admin.projects.project.offlineInputs.ReviewSection.importPDFFileTitle',
+    defaultMessage: 'Import scanned forms',
+  },
+  uploadPdfFile: {
+    id: 'app.containers.Admin.projects.project.offlineInputs.ImportModal.uploadPdfFile',
+    defaultMessage:
+      'Upload a <b>PDF file of scanned forms</b>. It must use a form printed from this phase. The form can be downloaded {hereLink}.',
```

Comment:
I know this was already there, but for accessibility the whole text of `The form can be downloaded here` should be the link.